docker/docker-compose.test.yml

```diff
@@ -0,0 +1,30 @@
+#
+# Copyright 2015-2019 The OpenZipkin Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+# in compliance with the License. You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software distributed under the License
+# is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+# or implied. See the License for the specific language governing permissions and limitations under
+# the License.
+#
+version: "3.6"
+services:
+  zipkin:
+    build:
+      context: ..
+      dockerfile: docker/Dockerfile
+    networks:
+      default:
+        aliases:
+        - zipkin
+  sut:
```

Comment:
perhaps `healthchecker` would be a better name?

Reply:
That name is hard coded as the name that docker hub checks for the test status - added a link to the docs so it's easier to follow.

Reply:
:+1: seems like an odd name but works for me